CONTRIBUTION.md

```diff
@@ -74,7 +74,20 @@ In general apply common sense to the Code you write. "Effective Java" is also al
 * enforce Immutability where possible
     * use ImmutableTypes for immutable values (`ImmutableList`, `ImmutableSet`, `ImmutableMap`, etc.)
     * avoid public properties
-    
+* Always use capital letters for format suffixes
+  * Bad format suffix: `0.5f`, `1000l`
+  * Good format suffiz: `0.5F`, `1000L`
+* Always use lower case letters for int literals
+  * Bad value: `0xFFF`, `0xBEEF`
+  * Good value: `0xfff`, `0xbeef`
+* Only use inner class then they are super short and closely coupled with the outer class. For example a 5 line inner class is fine but probably not a 20 line inner class.
+  * Only apply this to **class** and **enum**, *interfaces* and *annotations* should never be an inner.
```

Comment:
Annotations definetly may be inner classes think of nested annotations...
Interfaces maybe - though sometimes nested interfaces are useful...